### PR TITLE
CQ-4346472 Correct form path correlation in sling models.

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormStructureParserImpl.java [hunk 1]
```diff
@@ -26,18 +26,25 @@
     adapters = FormStructureParser.class)
 public class FormStructureParserImpl implements FormStructureParser {
 
+    private static final String FORM_CONTAINER_RESOURCE_TYPE = "core/fd/components/form/container/v2/container";
```
Comment (rismehta) [inline bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormStructureParserImpl.java:29]: We have a separate constants file for resource types, please re-use it from there.

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

**Comment by sakshi-arora1** [comment]
> -1 for now

Moved the constant.

**Review by rismehta — APPROVED** [review]
LGTM

**Pull request merged**